site/src/request_handlers/dashboard.rs

```diff
@@ -153,6 +155,8 @@ pub async fn handle_dashboard(ctxt: Arc<SiteCtxt>) -> ServerResult<dashboard::Re
                 ArtifactId::Commit(c) => format!("master: {}", &c.sha.to_string()[0..8]),
                 ArtifactId::Tag(aid) => aid.clone(),
             })
+            .collect::<Vec<_>>()
```

Comment:
Correct me if I'm wrong, but this would also affect the compile time benchmarks, and we definitely don't want to do that, because that would essentially skip compile time versions for which we do not have runtime data.

I think it would be better to return e.g. something like `Vec<Option<f64>>` for the runtime data, and then let the frontend skip the compiler versions for which there is no data available.